localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Places ###
+ city_ahn_qiraj:0 "City"
+ quarter_grim_batol:0 "Quarter"
+ city_grim_batol:0 "City"
 
-####################################################
-#Special titles
+ ### Horde ###
+ warchief_male:0 "Warchief"
+ warchief_female:0 "Warchief"
+
+ ### Scourge ###
+ lich_king_male:0 "Lich King"
+ lich_king_female:0 "Lich Queen"
+
+ ### Night Elves ###
  high_priest_kaldorei_male:0 "$HIGH_PRIEST_MALE_NAME$"
  high_priestess_kaldorei_female:0 "$HIGH_PRIEST_FEMALE_NAME$"
+
+ # Special cultural titles (sorted by culture, then government type).
+ ### Orcs ###
+ # Feudal
+ overlord_male_orc:0 "Overlord"
+ overlord_female_orc:0 "Overlord"
+ high_overlord_male_orc:0 "High Overlord"
+ high_overlord_female_orc:0 "High Overlord"
+ warlord_male_orc:0 "Warlord"
+ warlord_female_orc:0 "Warlord"
+ warchief_male_orc:0 "Warchief"
+ warchief_female_orc:0 "Warchief"
+
+ ### High Elves / Blood Elves ###
+ # Feudal
+ lord_male_highelf:0 "Lord"
+ lady_female_highelf:0 "Lady"
+ lordship_highelf:0 "Lordship"
+ high_king_male_highelf:0 "High King"
+ high_queen_female_highelf:0 "High Queen"
+ high_kingdom_highelf:0 "High Kingdom"
+
+ ### Highbornes / Nightbornes ###
+ # Nightborne specific
+ vicomte_male_nightborne:0 "Vicomte"
+ vicomtesse_male_nightborne:0 "Vicomtesse"
+ vicomte_nightborne:0 "Vicomte"
+
+ # Feudal
+ prince_male_highborne:0 "Prince"
+ princess_female_highborne:0 "Princess"
+ principality_highborne:0 "Principality"
+ grand_prince_male_highborne:0 "Grand Prince"
+ grand_princess_female_highborne:0 "Grand Princess"
+ grand_principality_highborne:0 "Grand Principality"
+ king_male_highborne:0 "King"
+ queen_female_highborne:0 "Queen"
+
+ ### lady_nightborne_female:0 "Lady"
+
+ ### Nagas ###
+ lord_male_naga:0 "Lord"
+ lady_female_naga:0 "Lady"
+ lordship_naga:0 "Lordship"
+ prince_male_naga:0 "Prince"
+ princess_female_naga:0 "Princess"
+ principalty_naga:0 "Principalty"
+ warlord_male_naga:0 "Warlord"
+ warlady_female_naga:0 "Warlady"
+ raj_male_naga:0 "Raj"
+ queen_female_naga:0 "Queen"
+
+ ### Humans group / Dwarf group ###
+ # Feudal
+ high_king_male:0 "High King"
+ high_queen_female:0 "High Queen"
+ high_kingdom:0 "High Kingdom"
+
+ # Republic
+ magistrate_male:0 "Magistrate"
+ magistrate_female:0 "Magistrate"
+ magistracy:0 "Magistracy"
+
+ ### Forsaken ###
+ # Republic
```

Comment:
Why is this on Republic? There's only feudal Forsaken in the game.

Reply:
And ? Would that prevent the forsakens from having republics? Magistrate forsakens exists: https://wowpedia.fandom.com/wiki/Magistrate_Sevren